scripts/run-unit-tests.sh

```diff
@@ -30,7 +30,12 @@ done
 
 SKIP="$SKIP -DFLB_WITHOUT_flb-it-fstore=1"
 # If no v6, disable that test
-[[ ! $(ip a) =~ ::1 ]] && SKIP="$SKIP -DFLB_WITHOUT_flb-it-network=1"
+if [ "$(uname)" = "Darwin" ]
```

Comment:
Why don't we just see if `ifconfig` exists and use it, otherwise use `ip` ? (Or vice versa as I think `ifconfig` is deprecated). That way we do not have to worry about specific uname's etc.

Reply:
Hmm..., I think that trying `ip` first is better.

Reply:
Yup, sounds good me. I don't think we should add specific tests for arch/OS/etc. unless we really need to.